Tasks/AzureTestPlanV0/Invokers/jestinvoker.ts

```diff
@@ -1,14 +1,39 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import tr = require("azure-pipelines-task-lib/toolrunner");
+import { executeJestCommand } from '../testLibExecutor';
 
+//Jest command like >set JEST_JUNIT_OUTPUT_NAME=TEST-Jest0-junit.xml
+//>npx jest --ci --reporters=default --reporters=jest-junit -t "JestTestName"
 export async function executeJestTests(testsToBeExecuted: string[]): Promise<number> {
 
-    // jest execution will be added
-    /*executable = npm
-    args = test
-    */
+    let finalStatus = 0; 
+    let jestNpm = tl.which("npm", true);
```

Comment:
why jestNpm?

Reply:
Just to know that this package will be used to install jest package.

Reply:
Changed to npmPath.